### ISSUE TITLE
tst_specific_filters blosc failure with latest git on Fedora Rawhide

### ISSUE THREAD

**Issue by opoplawski** [Unidata/netcdf-c#2572]
Building current head of main branch on Fedora Rawhide x86_64 with  blosc-1.21.1
```
FAIL: tst_specific_filters
==========================

findplugin.sh loaded
*** Testing processing of filter deflate for map nc
*** Testing processing of filter szip for map nc
*** Testing processing of filter bzip2 for map nc
*** Testing processing of filter blosc for map nc
10c10
<               ivar:_Filter = "32001,0,0,4,256,5,1,1" ;
---
>               ivar:_Filter = "32001,2,2,4,256,5,1,1" ;
14c14
<               fvar:_Filter = "32001,0,0,4,256,5,1,1" ;
---
>               fvar:_Filter = "32001,2,2,4,256,5,1,1" ;
FAIL tst_specific_filters.sh (exit status: 1)
```

**Comment by DennisHeimbigner** [comment]
What blosc package are you using? c-blosc?

**Comment by opoplawski** [comment]
Yeah, it seems to be this: https://github.com/Blosc/c-blosc

**Comment by DennisHeimbigner** [comment]
The list of values: 0,0,4,256,5,1,1 vs 2,2,4,256,5,1,1
represents the filter arguments used by the blosc wrapper (H5Zblosc.c,  I think).
They are defined as follows:
HDF5 Blosc filter takes 7 parameters as follows:
param[0] -- FILTER_BLOSC_VERSION (ignored)
param[1] -- BLOSC_VERSION_FORMAT (ignored)
param[2] -- typesize in bytes
param[3] -- chunksize in bytes (ignored)
param[4] -- compression level
param[5] -- shuffle enabled
param[6] -- compressor to use

So the problem is that the versions may have changed for the c-blosc
you used. I think I will solve this by suppressing the first two arguments
in the list.

**Comment by opoplawski** [comment]
This is still an issue with 4.9.3-rc1

**Comment by opoplawski** [comment]
It appears that this is fixed in 4.9.3-rc2
